tinkerbell/stack/templates/hook.yaml

```
@@ -1,4 +1,5 @@
 {{- if .Values.stack.hook.enabled }}
+{{- if .Values.stack.hook.downloadEnabled }}
```

you could combine both these into a single statement.
```suggestion
{{- if and .Values.stack.hook.enabled .Values.stack.hook.downloadEnabled }}
```